app/scripts/modules/amazon/src/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -190,5 +204,110 @@ describe('awsServerGroupCommandBuilder', function () {
 
       expect(command.tags).toEqual({ 'some-key': 'some-value' });
     });
+
+    it('sets unlimitedCpuCredits to false when building from source server group with standard credits', function () {
+      spyOn(this.instanceTypeService, 'getCategoryForInstanceType').and.returnValue(this.$q.when('selectedProfile'));
+
+      const baseServerGroup = {
```

Comment:
There is a `mockServerGroup` in the mock package that you can build off of it you want to reduce the boilerplate needed for a test.

Reply:
sure, will do.

Reply:
https://github.com/spinnaker/deck/pull/8752 @caseyhebebrand